### PR TITLE
[Module] Compute centroids module.

### PR DESCRIPTION
**Pull request by gagnonanthony**
## Describe your changes

Add compute centroids as a module.

## List test packages used by your module

- `bundles.zip`

## Checklist before requesting a review

- Create the tool:
  - [x] Edit `./modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./modules/nf-scil/<category>/<tool>/meta.yml`
- Generate the tests:
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/nextflow.config`
  - [x] Add test data locally for tests with the fork repository
  - [x] Generate the test infrastructure and _md5sum_ for all outputs
- Ensure the syntax is correct :
  - [x] Check indentation abides with the rest of the library (don't hesitate to correct others !)
  - [x] Lint everything. Ensure your variables have good names.

### CODE REVIEW

File: modules/nf-scil/bundle/centroid/main.nf
Comment (AlexVCaron) [inline modules/nf-scil/bundle/centroid/main.nf:1]: This should be split in two different modules : one for computing the centroids on bundles, the other for `TRACTOGRAM_RESAMPLE`.

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
Excellent job ! However, I prefer a different module for tractogram resampling and use it for centroids resampling in the future subworkflow, instead of putting it all in one module. Implemented the way it is here, it doesn't save us anything at execution.

File: modules/nf-scil/bundle/centroid/main.nf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.2.sif':
+        'scilus/scilus:2.0.2' }"
+
+    input:
+        tuple val(meta), path(bundles)
+
+    output:
+        tuple val(meta), path("*_centroid_{*}.{trk,tck}")   , emit: centroids
```
Comment (arnaudbore) [inline modules/nf-scil/bundle/centroid/main.nf:13]: you can use tck if a reference has been provided. Use only trk.

**Review by arnaudbore — APPROVED** [review]
GTG (almost)

**Pull request merged**